LocationPicker/LocationPickerViewController.swift

```diff
@@ -232,7 +232,7 @@ open class LocationPickerViewController: UIViewController {
 	}
 	
 	func showCoordinates(_ coordinate: CLLocationCoordinate2D, animated: Bool = true) {
-		let region = MKCoordinateRegionMakeWithDistance(coordinate, resultRegionDistance, resultRegionDistance)
+		let region = MKCoordinateRegion.init(center: coordinate, latitudinalMeters: resultRegionDistance, longitudinalMeters: resultRegionDistance)
```

Comment:
Can you remove `.init` there, so it's: `MKCoordinateRegion(center: coordinate...`